docs/docfx.json

```diff
@@ -40,6 +40,12 @@
           "**/*.png",
           "**/*.svg"
         ]
+      },
+      { // copy over example file
+        "files": [
+          "libs/Matchmaking/examples/P2PChat/P2PChat.cs"
+        ],
```

Comment:
What I'd like to do here is linking to the file on GitHub, but a relative link will not be handled by DocFx, and an absolute link to github.com/tree/etc would be bound to a specific path/branch. So I am doing the next best thing which is copying the linked file in the DocFx package, so at least the link will work.